internal/repository/memory/repository.go

```diff
@@ -0,0 +1,53 @@
+package memory
```

Comment:
this could be simplified by calling the package `repository` and moving the file to `internal/repository/memory.go`, so in future mongo.go and postgres.go could also be added.